docs/connect-blockchain/immutablex.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Integrate Web3Auth with the ImmutableX Blockchain
+sidebar_label: ImmutableX
+image: "/docs/contents/guides/banners/algorand.png"
```

Comment:
Please update the image here with the immutable one.

Reply:
done